### PR TITLE
Include Using Statement

### PR DESCRIPTION
**Pull request by spottedmahn**

### CODE REVIEW

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/modeling/relational/tables.md](https://github.com/spottedmahn/EntityFramework.Docs/blob/patch-2/entity-framework/core/modeling/relational/tables.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C16%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201703161355112025-369%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: entity-framework/core/modeling/relational/tables.md [hunk 1]
```diff
@@ -81,3 +81,6 @@ You can also specify a schema that the table belongs to.
         modelBuilder.Entity<Blog>()
             .ToTable("blogs", schema: "blogging");
 ````
+
+## Using Statement
+using System.ComponentModel.DataAnnotations.Schema;
```
Comment (tdykstra) [inline entity-framework/core/modeling/relational/tables.md]: @divega Is DataAnnotations.Schema needed for the attribute code only / what's needed for the fluent API code?

Comment (divega) [inline entity-framework/core/modeling/relational/tables.md]: It is only needed for the attribute to resolve.

Comment (divega) [inline entity-framework/core/modeling/relational/tables.md]: I believe for the rest of the code only `using Microsoft.EntityFrameworkCore;` is needed.

**Comment by bradygaster** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/core/modeling/relational/tables.md](https://github.com/spottedmahn/EntityFramework.Docs/blob/patch-2/entity-framework/core/modeling/relational/tables.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C4%5C10%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201704102102019859-369%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**